tests/bud.bats

```diff
@@ -2509,3 +2509,47 @@ _EOF
 #  run_buildah run $cid arch
 #  expect_output --substring "aarch64"
 }
+
+@test "bud with --manifest flag new manifest" {
+  _prefetch alpine
+  mytmpdir=${TESTDIR}/my-dir
+  mkdir -p ${mytmpdir}
+cat > $mytmpdir/Containerfile << _EOF
+from alpine
+run echo hello
+_EOF
+
+  run_buildah bud -q --manifest=testlist -t arch-test --signature-policy ${TESTSDIR}/policy.json ${mytmpdir} <<< input
```

Comment:
is the `<<< input` at the end here correct?  I'm not seeing a `--stdin` option.

Reply:
Yes this came from other tests. Not sure what it does.

Reply:
It must just be passing `input` in as input into the build command and then ignoring it.
Weird, but OK and the tests are happy.